packages/core/admin/ee/admin/content-manager/pages/EditView/InformationBox/components/AssigneeSelect/AssigneeSelect.js

```diff
@@ -72,77 +86,62 @@ export function AssigneeSelect() {
     }
   );
 
-  const formattedError =
-    (isError &&
-      formatMessage({
-        id: 'content-manager.reviewWorkflows.assignee.error',
-        defaultMessage: 'An error occurred while fetching users',
-      })) ||
-    (mutation.error && formatAPIError(mutation.error));
-
   return (
-    <Field error={formattedError} name={ASSIGNEE_ATTRIBUTE_NAME} id={ASSIGNEE_ATTRIBUTE_NAME}>
+    <Field name={ASSIGNEE_ATTRIBUTE_NAME} id={ASSIGNEE_ATTRIBUTE_NAME}>
       <Flex direction="column" gap={2} alignItems="stretch">
-        <FieldLabel>
-          {formatMessage({
+        <Combobox
+          clearLabel={formatMessage({
+            id: 'content-manager.reviewWorkflows.assignee.clear',
+            defaultMessage: 'Clear assignee',
+          })}
+          error={
+            (isError &&
+              canReadUsers &&
+              formatMessage({
+                id: 'content-manager.reviewWorkflows.assignee.error',
+                defaultMessage: 'An error occurred while fetching users',
+              })) ||
+            (mutation.error && formatAPIError(mutation.error))
+          }
+          disabled={!isLoading && users.length === 0}
+          name={ASSIGNEE_ATTRIBUTE_NAME}
+          id={ASSIGNEE_ATTRIBUTE_NAME}
+          value={currentAssignee ? currentAssignee.id : null}
+          onChange={(value) => handleChange({ value })}
+          onClear={() => handleChange({ value: null })}
+          placeholder={formatMessage({
+            id: 'content-manager.reviewWorkflows.assignee.placeholder',
+            defaultMessage: 'Select',
+          })}
```

Comment:
@jhoward1994 @lucasboilly I couldn't find anything in figma: should this be "Select" or "Select assignee"?

Reply:
I'd vote for "Select..." as assignee is stated above